src/services/NonFungibleTokens.js

```diff
@@ -1,7 +1,7 @@
 import sendJson from '../tmp_fetch_send_json';
 import { ACCOUNT_HELPER_URL, wallet } from '../utils/wallet';
 
-const TOKENS_PER_PAGE = 50;
+export const TOKENS_PER_PAGE = 2;
```

Comment:
2 at a time seems very low and would frustrate me from the users perspective... @corwinharrell was this a hard requirement for the design, or are you OK with us increasing this to something a bit larger like 4 or 8?

Reply:
Not a hard requirement! Let's increase to a default of 4

Reply:
Done.